Enhance MentionsList styling and functionality

adianasct opened the pull request:
- Adjusted the positioning and dimensions of the MentionsList component for improved layout.
- Added custom scrollbar styles for better user experience.
- Introduced a Handle component for visual indication of the MentionsList position.
- Updated MemberItem styling for consistency and improved usability.

adianasct committed:
Enhance MentionsList styling and functionality

- Adjusted the positioning and dimensions of the MentionsList component for improved layout.
- Added custom scrollbar styles for better user experience.
- Introduced a Handle component for visual indication of the MentionsList position.
- Updated MemberItem styling for consistency and improved usability.

adianasct committed:
Update MentionsList max-height for improved visibility

karensct commented on src/components/SendMessageInput/MentionsPlugin/index.tsx:
```diff
@@ -424,27 +425,64 @@ const MentionsList = styled.ul<{
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background: #37393c;
```
this is hardcoded value

karensct commented on src/components/SendMessageInput/MentionsPlugin/index.tsx:
```diff
@@ -424,27 +425,64 @@ const MentionsList = styled.ul<{
+  }
+
+  &::-webkit-scrollbar-thumb:hover {
+    background: #37393c;
```
this is hardcoded value

karensct commented on src/components/SendMessageInput/MentionsPlugin/index.tsx:
```diff
@@ -302,6 +302,7 @@ function MentionsContainer({
           />
         ))}
       </MentionsList>
+      <Handle backgroundColor={background} />
```
this part and all mentioned popup BG is not right

<img width="1075" height="632" alt="image" src="https://github.com/user-attachments/assets/10685298-e2c4-4f5d-b022-25413318a96f" />


and check image above see isssue which you have

check all with all possible cases and make it like figma(pixel perfect)
do not use hardcoded valuse like hardcoded color or hardcoded width height

karensct commented on src/components/SendMessageInput/MentionsPlugin/index.tsx:
```diff
@@ -424,27 +425,64 @@ const MentionsList = styled.ul<{
 export const MemberItem = styled.li<{ isActiveItem?: boolean; activeBackgroundColor: string }>`
   display: flex;
   align-items: center;
+  width: 340px;
```
I mean this for hardcoded width and height

adianasct committed:
Enhance MentionsContainer and MentionsList styling

- Added scrollbar color customization to the MentionsList for improved aesthetics.
- Updated the MentionsContainer to include the new scrollbar color in its props.
- Adjusted MemberItem width to a maximum value for better layout consistency.

karensct commented on src/components/SendMessageInput/MentionsPlugin/index.tsx:
```diff
@@ -237,7 +237,11 @@ function MentionsContainer({
+  const {
+    [THEME_COLORS.BORDER]: borderColor,
+    [THEME_COLORS.BACKGROUND]: background,
+    [THEME_COLORS.SURFACE_1]: scrollbarThumbColor
```
see what colors used in other  places for &::-webkit-scrollbar {
and see image issue
<img width="441" height="181" alt="image" src="https://github.com/user-attachments/assets/f2ffb11d-7320-4bd2-8cd8-578b605878bd" />

karensct committed:
Merge branch 'fix/disappearing-messages-hiden-issue' of https://github.com/sceyt/sceyt-chat-react-uikit into fix/mentions-popup-restyle

karensct committed:
Merge branch 'fix/disappearing-messages-hiden-issue' of https://github.com/sceyt/sceyt-chat-react-uikit into fix/mentions-popup-restyle

karensct committed:
Implement dynamic alignment for mentions list based on overflow detection

- Added state management to determine if the mentions list should align to the right to prevent overflow.
- Introduced a useEffect hook to check for overflow on mount and when options change, adjusting alignment accordingly.
- Updated MentionsList styled component to conditionally set left and right positioning based on alignment state.

karensct committed:
Add message removal on channel switch in saga

- Integrated `removeAllMessages` call in the `switchChannel` saga to clear messages when switching channels.
- Ensured that the active channel is updated correctly after clearing messages, improving channel transition handling.
